packages/core/src/State.ts

```diff
@@ -53,7 +55,8 @@ interface MachineSnapshotBase<
   TTag extends string,
   TOutput,
   TMeta,
-  _TUnusedButLeftForCompatReasons = never
+  _TUnusedButLeftForCompatReasons = never,
```

Comment:
it's really hard to judge those things but it's probably fine to remove this one now

Reply:
[5a3404d](https://github.com/statelyai/xstate/pull/4979/commits/5a3404dc0e7250e0a9b861287235121fe45a5f95)